src/StdStyle.sol

```diff
@@ -0,0 +1,241 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.22 <0.9.0;
+
+import {Vm} from "./Vm.sol";
+
+library StdStyle {
+    Vm private constant vm = Vm(address(uint160(uint256(keccak256("hevm cheat code")))));
+
+    string constant RED = "\u001b[91m";
+    string constant GREEN = "\u001b[92m";
+    string constant YELLOW = "\u001b[93m";
+    string constant BLUE = "\u001b[94m";
+    string constant MAGENTA = "\u001b[95m";
+    string constant CYAN = "\u001b[96m";
+    string constant BOLD = "\u001b[1m";
+    string constant DIM = "\u001b[2m";
+    string constant ITALIC = "\u001b[3m";
+    string constant UNDERLINE = "\u001b[4m";
+    string constant REVERSE = "\u001b[7m";
+    string constant RESET = "\u001b[0m";
+
+    function red(string memory self) internal view returns (string memory) {
+        return string.concat(RED, self, RESET);
+    }
+
+    function red(uint256 self) internal view returns (string memory) {
+        return string.concat(RED, vm.toString(self), RESET);
```

Comment:
Why dont you just do:
```solidity
    function red(string memory self) internal view returns (string memory) {
        return string.concat(RED, self, RESET);
    }

    function red(uint256 self) internal view returns (string memory) {
        return red(vm.toString(self));
    }
```

That way in the future we only ever have to change the string implementation if we every want to change the logic, and all the others are just a conversion to string then the `red(string)` function

Reply:
Yeah, this is a better approach thanks for the suggestion @brockelmore. I've changed this in the last commit.